builtin/plugins/address_mapper/address_mapper.go

```diff
@@ -47,13 +58,27 @@ var (
 	// address mapper contract.
 	ErrAlreadyRegistered = errors.New("[Address Mapper] identity mapping already exists")
 
-	AddressPrefix = "addr"
+	AddressPrefix    = "addr"
+	NoncePrefix      = "np"
+	SupportedChainID = []string{"eth", "tron", "binance"}
 )
 
 func addressKey(addr loom.Address) []byte {
 	return util.PrefixKey([]byte(AddressPrefix), addr.Bytes())
 }
 
+func nonceKey(addr loom.Address) []byte {
+	return util.PrefixKey([]byte(NoncePrefix), addr.Bytes())
+}
+
+func multiChainAddressKey(addr loom.Address, targetChainID string, sequence uint64) []byte {
+	return util.PrefixKey([]byte(AddressPrefix), addr.Bytes(), []byte(targetChainID), UintToBytesBigEndian(sequence))
+}
+
+func getMultiChainAddressKey(addr loom.Address, targetChainID string) []byte {
+	return util.PrefixKey([]byte(AddressPrefix), addr.Bytes(), []byte(targetChainID), []byte{})
```

Comment:
Should just be `util.PrefixKey([]byte(AddressPrefix), addr.Bytes(), []byte(targetChainID))`